### PR TITLE
add Enter your name

### PR DESCRIPTION
**Pull request by daviduntalan**
Wanted the add Enter your name?

### CODE REVIEW

**Review by ladylou-ph — CHANGES_REQUESTED** [review]
Thanks for this but can you add Enter your location

File: ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Enter your name:");
```
Comment (ladylou-ph) [inline ConsoleApp1/Program.cs]: This is cool, i never thought of this before. But can you add Enter your location?

Comment (ladylou-ph) [inline ConsoleApp1/Program.cs:10]: This is cool, i never thought of this before. 
but can you add something more here.

**Review by ladylou-ph — CHANGES_REQUESTED** [review]
can you add Enter your location?

File: ConsoleApp1/Program.cs (hunk 1 above)
Comment (daviduntalan) [reply ConsoleApp1/Program.cs:10]: Sure!

Comment (ladylou-ph) [reply ConsoleApp1/Program.cs:10]: Thanks.

**Review by ladylou-ph — APPROVED** [review]

**Pull request merged**